{
  "active_lock_reason": null,
  "assignee": null,
  "assignees": [],
  "author_association": "CONTRIBUTOR",
  "body": "\r\n**What happened**:\r\nWhen running conformance test suite using ci-kubernetes-conformance-image job in the CI process the following test fails as it tries to calculate the number of worker nodes incorrectly.\r\n[sig-apps] Daemon set [Serial] should rollback without unnecessary restarts [Conformance] \r\n\r\n**What you expected to happen**:\r\nThe test must read the number nodes correctly so that the checks in the test will pass\r\n/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/apps/daemon_set.go:425\r\n\r\n**How to reproduce it (as minimally and precisely as possible)**:\r\nRun the  ci-kubernetes-conformance-image in a docker container to initiate the 2 node cluster using ./../test-infra/experiment/kind-conformance-e2e.sh\r\n\r\n**Environment**:\r\n- Kubernetes version (use `kubectl version`): 1.14\r\n- Cloud provider or hardware configuration: amd64\r\n\r\n/sig testing\r\n/area conformance\r\n/kind test\r\n/release-note-none\r\n/sig apps",
  "closed_at": "2019-03-22T07:04:31Z",
  "closed_by": {
    "avatar_url": "https://avatars0.githubusercontent.com/u/20407524?v=4",
    "events_url": "https://api.github.com/users/k8s-ci-robot/events{/privacy}",
    "followers_url": "https://api.github.com/users/k8s-ci-robot/followers",
    "following_url": "https://api.github.com/users/k8s-ci-robot/following{/other_user}",
    "gists_url": "https://api.github.com/users/k8s-ci-robot/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/k8s-ci-robot",
    "id": 20407524,
    "login": "k8s-ci-robot",
    "node_id": "MDQ6VXNlcjIwNDA3NTI0",
    "organizations_url": "https://api.github.com/users/k8s-ci-robot/orgs",
    "received_events_url": "https://api.github.com/users/k8s-ci-robot/received_events",
    "repos_url": "https://api.github.com/users/k8s-ci-robot/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/k8s-ci-robot/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/k8s-ci-robot/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/k8s-ci-robot"
  },
  "comments": 0,
  "comments_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/75348/comments",
  "created_at": "2019-03-14T00:05:34Z",
  "events_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/75348/events",
  "html_url": "https://github.com/kubernetes/kubernetes/issues/75348",
  "id": 420774228,
  "labels": [
    {
      "color": "0052cc",
      "default": false,
      "description": "Issues or PRs related to kubernetes conformance tests",
      "id": 717986501,
      "name": "area/conformance",
      "node_id": "MDU6TGFiZWw3MTc5ODY1MDE=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/area/conformance"
    },
    {
      "color": "e11d21",
      "default": false,
      "description": "Categorizes issue or PR as related to a bug.",
      "id": 105146071,
      "name": "kind/bug",
      "node_id": "MDU6TGFiZWwxMDUxNDYwNzE=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/kind/bug"
    },
    {
      "color": "d2b48c",
      "default": false,
      "description": "Categorizes an issue or PR as relevant to SIG Apps.",
      "id": 404091735,
      "name": "sig/apps",
      "node_id": "MDU6TGFiZWw0MDQwOTE3MzU=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/sig/apps"
    },
    {
      "color": "d2b48c",
      "default": false,
      "description": "Categorizes an issue or PR as relevant to SIG Testing.",
      "id": 483069764,
      "name": "sig/testing",
      "node_id": "MDU6TGFiZWw0ODMwNjk3NjQ=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/sig/testing"
    }
  ],
  "labels_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/75348/labels{/name}",
  "locked": false,
  "milestone": null,
  "node_id": "MDU6SXNzdWU0MjA3NzQyMjg=",
  "number": 75348,
  "performed_via_github_app": null,
  "repository_url": "https://api.github.com/repos/kubernetes/kubernetes",
  "state": "closed",
  "title": "Daemon Set Conformance test fails on Kind Cluster using ci-kubernetes-conformance-image job.",
  "updated_at": "2019-03-22T07:04:31Z",
  "url": "https://api.github.com/repos/kubernetes/kubernetes/issues/75348",
  "user": {
    "avatar_url": "https://avatars3.githubusercontent.com/u/5403596?v=4",
    "events_url": "https://api.github.com/users/brahmaroutu/events{/privacy}",
    "followers_url": "https://api.github.com/users/brahmaroutu/followers",
    "following_url": "https://api.github.com/users/brahmaroutu/following{/other_user}",
    "gists_url": "https://api.github.com/users/brahmaroutu/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/brahmaroutu",
    "id": 5403596,
    "login": "brahmaroutu",
    "node_id": "MDQ6VXNlcjU0MDM1OTY=",
    "organizations_url": "https://api.github.com/users/brahmaroutu/orgs",
    "received_events_url": "https://api.github.com/users/brahmaroutu/received_events",
    "repos_url": "https://api.github.com/users/brahmaroutu/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/brahmaroutu/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/brahmaroutu/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/brahmaroutu"
  }
}